<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="OmeE" failures="0" tests="2" name="Default test" time="0.23" errors="0" timestamp="2023-07-26T19:44:11 IST">
  <testcase classname="TestComponents.BaseTest" name="@BeforeTest launchApplication" time="0.124">
    <failure type="org.openqa.selenium.remote.NoSuchDriverException" message="msedgedriver located at D:\Drivers\edge.exe, but invalid
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors/driver_location/
Build info: version: &amp;apos;4.10.0&amp;apos;, revision: &amp;apos;c14d967899&amp;apos;
System info: os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;17.0.6&amp;apos;
Driver info: driver.version: EdgeDriver">
      <![CDATA[org.openqa.selenium.remote.NoSuchDriverException: msedgedriver located at D:\Drivers\edge.exe, but invalid
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors/driver_location/
Build info: version: '4.10.0', revision: 'c14d967899'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.6'
Driver info: driver.version: EdgeDriver
at org.openqa.selenium.remote.service.DriverFinder.getPath(DriverFinder.java:45)
at org.openqa.selenium.edge.EdgeDriver.generateExecutor(EdgeDriver.java:70)
at org.openqa.selenium.edge.EdgeDriver.<init>(EdgeDriver.java:59)
at org.openqa.selenium.edge.EdgeDriver.<init>(EdgeDriver.java:55)
at org.openqa.selenium.edge.EdgeDriver.<init>(EdgeDriver.java:47)
at org.openqa.selenium.edge.EdgeDriver.<init>(EdgeDriver.java:43)
at TestComponents.BaseTest.initializeDriver(BaseTest.java:40)
at TestComponents.BaseTest.launchApplication(BaseTest.java:50)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:637)
at org.testng.TestRunner.beforeRun(TestRunner.java:627)
at org.testng.TestRunner.run(TestRunner.java:589)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeTest launchApplication -->
  <testcase classname="TestComponents.BaseTest" name="@AfterMethod tearDown" time="-1.690380850828E9">
    <skipped/>
  </testcase> <!-- @AfterMethod tearDown -->
  <testcase classname="TestComponents.BaseTest" name="@AfterMethod tearDown" time="-1.690380850848E9">
    <skipped/>
  </testcase> <!-- @AfterMethod tearDown -->
  <testcase classname="common.project.ErrorValidationTest" name="LoginErrorValidation" time="0.0">
    <skipped/>
  </testcase> <!-- LoginErrorValidation -->
  <testcase classname="common.project.ErrorValidationTest" name="submitOrder" time="0.0">
    <skipped/>
  </testcase> <!-- submitOrder -->
</testsuite> <!-- Default test -->
